aws/credentials/processcreds/provider.go

```diff
@@ -0,0 +1,421 @@
+/*
+Package processcreds is a credential Provider to retrieve `credential_process`
+credentials.
+
+**Warning:**
+The following describes a method of sourcing credentials from an external
```

Comment:
Nit changing this to the style below. Go doc doesn't provide any highlighting for markdown bold syntax.

```
WARNING: The following describes a method of sourcing....
````